fileWithPRComments.js

```diff
@@ -0,0 +1,8 @@
+/* global QamineUtils */
+
+"use strict";
+
+(function($, window) {
+  $('.jsSelectedMainEmail').text(postVariable.email);
```

Comment:
![Codacy](http://release.k8s.dev.codacy.org/assets/images/favicon.png) Issue found: ['postVariable' is not defined.](http://release.k8s.dev.codacy.org/manual/devk8scodacytestuser/CheckPrCommentsPublicProjectDev/pullRequest?prid=189)